editor/gui/editor_dir_dialog.cpp

```diff
@@ -99,6 +116,10 @@ void EditorDirDialog::reload(const String &p_path) {
 void EditorDirDialog::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_ENTER_TREE: {
+			if (!FileSystemDock::get_singleton()->is_connected("folder_color_changed", callable_mp(this, &EditorDirDialog::reload))) {
```

Comment:
Is the condition necessary? The connection below doesn't have one. Also if you want to make sure it's not already connected, use `NOTIFICATION_READY`.